### PR TITLE
added Address validations

### PR DESCRIPTION
**Pull request by ktarun1419**

### CODE REVIEW

**Review by ermvrs — APPROVED** [review]

File: src/utils/validations.ts [hunk 1]
```diff
@@ -1,10 +1,28 @@
+import { addHexPrefix, hexPadding, removeHexPrefix } from './padding'
+
 /* eslint-disable @typescript-eslint/no-unused-vars */
```
Comment (ermvrs) [inline src/utils/validations.ts:3]: Can you remove eslint disable line?

**Review by ermvrs — CHANGES_REQUESTED** [review]
Can you remove eslint disable

File: src/utils/validations.ts (hunk 1 above)
Comment (ktarun1419) [reply src/utils/validations.ts:3]: yes sure

**Review by ermvrs — APPROVED** [review]

**Comment by LucasLvy** [comment]
@all-contributors please add @ktarun1419 code, test

**Pull request merged**